docs-en/chapter_preface/summary.md

```diff
@@ -1,8 +1,8 @@
 # Summary
 
-- The main audience of this book is beginners in algorithm. If you already have some basic knowledge, this book can help you systematically review your algorithm knowledge, and the source code in this book can also be used as a "brushing tool library".
+- The main audience of this book is beginners in algorithm. If you already have some basic knowledge, this book can help you systematically review your algorithm knowledge, and the source code in this book can also be used as a "Coding Toolkit".
 - The book consists of three main sections, Complexity Analysis, Data Structures, and Algorithms, covering most of the topics in the field.
-- For newcomers to algorithms, it is crucial to read an introductory book in the beginning stages to take the road less traveled.
+- For newcomers to algorithms, it is crucial to read an introductory book in the beginning stages to avoid many detours or common pitfalls.
 - Animations and graphs within the book are usually used to introduce key points and difficult knowledge. These should be given more attention when reading the book.
```

Comment:
Chinese：书内的动画和图解通常用于介绍重点和难点知识。阅读本书时，应给予这些内容更多关注。
Animations and graphs should be given more attention as they are used to introduce key points and difficult knowledge.

English is an explicit and upfront language. I think it's better to place the key part of the sentence at the beginning. And since there is a context, I delete "within the book".

Reply:
我也认为这个部分在这里语意不通顺，但我认为可以先按照原文翻译，尽量不改动，除非有错误。

I also believe that this part is not clear in the context of this sentence, but I think we should translate it as close to the original text as possible, unless there are errors.